sql/glam-fenix-dev/glam_etl/firefox_desktop__clients_daily_histogram_aggregates_metrics_v1/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+friendly_name: |-
+  Glam Etl - Firefox Desktop - Clients Daily Histogram Aggregates Metrics
+description: |-
+  [DESCRIPTION_MISSING]
+owners:
+- data-platform-infra-wg@mozilla.com
```

Comment:
@edugfilho should we assign all the glam stuff to you as the owner?

Reply:
yes please. Thank you for taking care of this!

Reply:
Amazing, thank you @edugfilho 🙇